fastify/entry.sh

```diff
@@ -1,44 +1,20 @@
 #! /bin/bash -e
 # `CONFIG` - name of configuration file in s3
 #
-# 3 modes:
-# 1: agent from artifactory
-#  - `AGENT_VERSION` - e.g. 2.11.0
-#  - `AGENT_HASH` e.g. 20200220-2232.9aa9d178
-# 2: agent from s3 bucket:
-#  - `AGENT` - name of .tgz in s3://node-agents
-# 3: agent-less
-#  - only env var needed is `CONFIG`
+# 2 modes:
+# 1: agent from mounted volume
+#  - `AGENT` - boolean, will mount a dir to `/opt/contrast` to copy agent
+#  - `CONFIG` - which config from s3://node-agent-configs to use
+# 2: agent-less
 
 if [[ -n "$AGENT" ]];
```

Comment:
do we even need the AGENT environment variable anymore?  can't we just check if `/opt/contrast/node_contrast*.tgz` exists?